src/redux/books/books.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const Books = () => (
+  <div className="book-list">
+    <ul>
+      <li>
+        <span className="title">The Hunger Games</span>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <span className="title">Dune</span>
+        <button type="button">Remove</button>
+      </li>
+      <li>
+        <span className="title">Capital in the Twenty-First Century</span>
+        <button type="button">Remove</button>
+      </li>
+    </ul>
```

Comment:
- [ ] You hardcoded your Book whereas your book should be a separate component. Let's refactor this to be neater. Kindly create a new Book (or OneBook/SingleBook) component that represents **one**. The component should take in book data as a prop and render the `title` and `author` with a placeholder Remove button. Then render your books here instead using the created component.

  **NOTE:** Your components should be created in the `src/components` directory.